src/main/java/data/workouts/WorkoutList.java

```diff
@@ -290,10 +297,38 @@ public Workout deleteWorkout(String userArgument) throws InvalidWorkoutException
         workoutsDisplayList.remove(indexToDelete - 1);
         String deletedWorkoutKey = deletedWorkout.toString();
         getWorkoutsHashMapList().remove(deletedWorkoutKey);
+        ui.printDeleteWorkoutMessage(deletedWorkout);
+
+        String deletedWorkoutDetail = deletedWorkout.toString();
+        deletePlanContainsDeletedWorkout(deletedWorkoutDetail, planList);
```

Comment:
hmm is there a rationale to why the UI call is here instead of at the WorkoutCommand?

Reply:
` deletePlanContainsDeletedWorkout()` method will print the messages to show any plan has the deleted workout and was removed successfully. If we call the `printDeleteWorkoutMessage()` in workout command, the workout deleted message will be shown after the plan deleted message.

Reply:
hmm because i think it would be more consistent if everything is in the WorkCommand.java instead of putting ui printing and deleteplancontainsworkout method in the same method as the deleteWorkout...